cmds/ts.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const EPILOG = `
+Presets:
+\`check\`      Checks src and test folders for Typescript errors
```

Comment:
Will this mean we have to include TS comments on all of our tests?

Reply:
No, it works as is but it may output errors for some inferred types or defined types in the source code.
Checking the tests enables us to be the users of our own types and be able to identify issues in the types early.